graspy/utils/utils.py

```diff
@@ -221,30 +221,35 @@ def to_laplace(graph, form="DAD"):
     r"""
     A function to convert graph adjacency matrix to graph laplacian. 
 
-    Currently supports I-DAD and DAD laplacians, where D is the diagonal
+    Currently supports I-DAD, DAD, and RDAD laplacians, where D is the diagonal
     matrix of degrees of each node raised to the -1/2 power, I is the 
-    identity matrix, and A is the adjacency matrix
+    identity matrix, and A is the adjacency matrix.
+    
+    R-DAD is regularized laplacian: where D = D + t*I
+    t defaults to the average degree
 
     Parameters
     ----------
     graph: object
         Either array-like, (n_vertices, n_vertices) numpy array,
         or an object of type networkx.Graph.
 
-    form: {'I-DAD' (default), 'DAD'}, string, optional
+    form: {'I-DAD' (default), 'DAD', 'R-DAD'}, string, optional
         
         - 'I-DAD'
             Computes :math:`L = I - D*A*D`
         - 'DAD'
             Computes :math:`L = D*A*D`
+        - 'R-DAD'
+            Computes :math:`L = D_t*A*D_t` where D_t = D + t*I
```

Comment:
same thing here. Wrap `D_t = D + t*I` with `:math:`. 

Always take a look at the rendered website when you make changes to docstrings to see if looks right:
https://deploy-preview-125--graspy.netlify.com/reference/utils.html#graspy.utils.to_laplace

Reply:
updated